docs/section-4-syntax-highlighting.md

```diff
@@ -58,6 +58,25 @@ The Tree-sitter highlighting system works by annotating ranges of source code wi
 
 In your config file, the `"theme"` value is an object whose keys are dot-separated highlight names like `function.builtin` or `keyword`, and whose values are JSON expressions that represent text styling parameters.
 
+### Parse Theme
+
+The Tree-sitter `parse` command will output a pretty-printed CST when the `--cst` option is used. You can control which colors are used for various parts of the tree in your configuration file:
+
+```json
+{
+  "parse-theme": {
```

Comment:
Would it be a good idea to set one of these to `null` to indicate that assigning each field is optional?

Reply:
add a note instead to clearly convey that - `Note: setting the value to null will render the text without any color` or something along those lines

Reply:
Settled on using the default color for entries that are omitted or set to `null`.